### ISSUE TITLE
Windows version javascript error

### ISSUE THREAD

**Issue by Jon-guy30** [FreeTubeApp/FreeTube#105]
Hi man, this could be a very good program if it would only run for me. (I found it through a comment on a blog) Launching the windows version from the zip file (I assume it's portable?) results in a javascript error. Uncaught exception. I tried deleting the folder and it said one or more files is still used by the program. Then I went to the task manager and there it is, the process is running even though it gave an error? And the windows executable file gave me a smartscreen Windows protected your computer pop-up. I don't care for the executable that much, if you could fix the zip version that would be cool. 
![javascript error](https://user-images.githubusercontent.com/10241225/40977801-8cf7ce34-68c1-11e8-8dcf-79abba9a01c7.PNG)

![it s running](https://user-images.githubusercontent.com/10241225/40977829-97812058-68c1-11e8-9cd2-5afe8a06ff2b.PNG)

**Comment by LexHimself** [comment]
@PrestonN Should the compilers even be needed on an executable? 
What is noticed is that it refers to the es6-shim.js, should this not be init? 

Was able to reproduce it in Wine:
![2018-06-05-151834_357x397_scrot](https://user-images.githubusercontent.com/12938372/40978829-7c3b6bba-68d4-11e8-998e-58697baa9d01.png)

**Comment by Jon-guy30** [comment]
Yes, it occurs on the zip file. Setup gave an SS pop-up.

**Comment by PrestonN** [comment]
Hey there.  This looks to be because of some changing that I was trying to do during the build process.  In previous releases, I have been compressing the zip manually.  Electron-forge has an option to do this for me so I used that for this build.  It looks like something about their build process makes it error out.  I tested this on my Windows system and the manually zipped project worked while the automatic one errored out like above.  I will go back to manually zipping the project for future releases.  I will post a fixed link in the releases shortly.  I will keep this issue open in the mean time.
